packages/react-test-renderer/src/ReactShallowRenderer.js

```diff
@@ -384,7 +384,7 @@ class ReactShallowRenderer {
         'an infinite loop.',
     );
 
-    if (componentIdentity === this._currentlyRenderingComponent) {
+    if (componentIdentity === this._previousComponentIdentity) {
```

Comment:
Why is this change correct? It disagrees with what the comment inside says ("This is a render phase update"). What was it supposed to mean? Why was it changed?

Reply:
Yeah, I missed updating the comment =/
Before this change `_dispatchAction` ignored calls after the render returned, this checks if its the same component and queue the action.